src/styles/zuehlke.css

```
@@ -17,10 +17,16 @@
     --zuehlke-blue: #0099cc;
     --zuehlke-bright-green: #ccff00;
     --zuehlke-green: #00cc66;
+    --zuehlke-gradient: linear-gradient(to right bottom, var(--zuehlke-purple), var(--zuehlke-bright-blue));
```

Quickly checked the gradient in the the slidedecks master view - would be the following:

```suggestion
        --zuehlke-gradient: linear-gradient(to right bottom, var(--zuehlke-purple) 10%, #A1C2E6);
```

---

Updated it in https://github.com/webplatformz/zuehlke-reveal-package/pull/4 as well